packages/edit-site/src/components/global-styles/font-library-modal/font-collection.js

```diff
@@ -309,7 +311,11 @@ function FontCollection( { slug } ) {
 					<Spacer margin={ 4 } />
 
 					{ ! selectedCollection?.font_families && ! notice && (
-						<Spinner />
+						<Spacer paddingY={ 6 }>
+							<Flex justify="center" align="center">
```

Comment:
```suggestion
							<Flex justify="center">
```

The default value of the `align` prop is `center`, so I think it's okay without explicitly specifying it.

https://github.com/WordPress/gutenberg/tree/trunk/packages/components/src/flex/flex#align-csspropertiesalignitems